src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -125,84 +173,75 @@ protected override void OnNextCore(Block block)
         /// <para>
         /// Before relaying, verify the block is still in the best chain else discard it.
         /// </para>
+        /// <para>
         /// TODO: consider moving the relay logic to the <see cref="LoopSteps.ProcessPendingStorageStep"/>.
+        /// </para>
         /// </remarks>
-        public void RelayWorker()
+        private async Task SendBatchLockedAsync()
```

Comment:
What about cancellation? when node shouts down do we know how to wait for this method to finish?

Reply:
Fixed